package.json

```diff
@@ -22,34 +28,45 @@
     "array-range": "^1.0.1",
     "broccoli-file-creator": "^2.1.1",
     "broccoli-merge-trees": "^3.0.2",
-    "ember-auto-import": "^1.5.3",
-    "ember-cli-babel": "^6.6.0",
+    "ember-auto-import": "^2.4.2",
+    "ember-cli-babel": "^7.26.11",
     "merge-options": "^1.0.1",
     "pretender": "^3.2.0",
     "recursive-readdir-sync": "^1.0.6"
   },
   "devDependencies": {
+    "@ember/test-helpers": "^2.8.1",
+    "@embroider/test-setup": "^1.8.3",
+    "@glimmer/tracking": "^1.1.2",
     "broccoli-asset-rev": "^2.4.5",
-    "ember-cli": "^3.15.2",
-    "ember-cli-dependency-checker": "^2.0.0",
-    "ember-cli-eslint": "^4.2.1",
-    "ember-cli-htmlbars": "^2.0.1",
+    "ember-cli": "^4.6.0",
+    "ember-cli-dependency-checker": "^3.3.1",
+    "ember-cli-htmlbars": "^6.1.0",
     "ember-cli-htmlbars-inline-precompile": "^1.0.0",
-    "ember-cli-inject-live-reload": "^1.4.1",
+    "ember-cli-inject-live-reload": "^2.1.0",
     "ember-cli-qunit": "^4.4.0",
     "ember-cli-shims": "^1.2.0",
     "ember-cli-sri": "^2.1.0",
-    "ember-cli-uglify": "^2.0.0",
-    "ember-export-application-global": "^2.0.1",
-    "ember-load-initializers": "^1.0.0",
-    "ember-resolver": "^4.0.0",
-    "ember-source": "~2.18.0",
-    "eslint-plugin-ember": "^5.0.0",
-    "eslint-plugin-node": "^5.2.1",
-    "loader.js": "^4.2.3"
+    "ember-cli-terser": "^4.0.2",
+    "ember-load-initializers": "^2.1.2",
+    "ember-page-title": "^7.0.0",
```

Comment:
If we remove the `{{page-title}}` from the dummy app we can remove this dependency as well right? Not a biggie, but preferably if we don't need a dep then we don't add it - will leave this one up to you.

Reply:
The `add-on` doesn't need anything from the dummy application, as far as I can tell. The dev-dependencies won't be pulled in by the host-application and will only be used in the dummy-application, that is created as the default testing layer for a regular add-on.

Reply:
Ok no problem, I'm more of the "if we aren't using a dependency then don't depend on it" frame of mind. But as I said no biggie, I really don't mind either way.

On a related note, could we remove the jQuery resolution from the package.json file now that we have upgraded off of ember 2.18 (which used to use jQuery), and we should probably sync the package-lock.json in this PR also seeing as we are changing dependencies, wdyt?